src/Workspaces/Core/Portable/Extensions/ReportDiagnosticExtensions.cs

```diff
@@ -1,24 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis
-{
-    internal static class ReportDiagnosticExtensions
-    {
-        public static string ToEditorConfigString(this ReportDiagnostic reportDiagnostic)
```

Comment:
Moved to another existing extensions method file, which was moved to shared layer.